lib/main.dart

```diff
@@ -143,7 +147,54 @@ class _MyHomePageState extends State<MyHomePage> {
                 ? LoadingInfo(loading)
                 : IconButton(
                     icon: Icon(Icons.menu),
-                    onPressed: () => Scaffold.of(context).openDrawer(),
+                    //onPressed: () => Scaffold.of(context).openDrawer(),
+                    onPressed: () => showModalBottomSheet(
+                      context: context,
+                      shape: RoundedRectangleBorder(
+                        borderRadius: BorderRadius.only(
+                          topLeft: Radius.circular(12),
+                          topRight: Radius.circular(12),
+                        ),
+                      ),
+                      builder: (context) => Container(
+                        padding: EdgeInsets.only(top: 8),
```

Comment:
[nit] This and some other things here could be `const`. I don't think we are being consistent with this in `hn_app`, so no big deal.

Reply:
adding it anyway :)